it.emulab.mamedev.clrmamepro/it.emulab.mamedev.clrmamepro.yaml

```diff
@@ -0,0 +1,112 @@
+      - ln -s /app/lib/32bit/lib/ld-linux.so.2 /app/lib/ld-linux.so.2
+
+  - name: setup-wine
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/wine
+
+  - name: setup-wine-compat32
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/wine-32bit
+
+  - name: clrmamepro
+    buildsystem: simple
+    build-commands:
+      - install -d /app/bin
+      - install apply_extra /app/bin
+      - install clrmamepro /app/bin
+      - install -Dm644 it.emulab.mamedev.clrmamepro.appdata.xml /app/share/appdata/it.emulab.mamedev.clrmamepro.appdata.xml
+      - install -Dm644 it.emulab.mamedev.clrmamepro.desktop /app/share/applications/it.emulab.mamedev.clrmamepro.desktop
+      - install -Dm644 it.emulab.mamedev.clrmamepro.48x48.png /app/share/icons/hicolor/48x48/apps/it.emulab.mamedev.clrmamepro.png
+      - install -Dm644 it.emulab.mamedev.clrmamepro.32x32.png /app/share/icons/hicolor/32x32/apps/it.emulab.mamedev.clrmamepro.png
+    sources:
+    - type: extra-data
+      only-arches:
+        - x86_64
+      url: https://mamedev.emulab.it/clrmamepro/binaries/cmp4034_64.zip
+      sha256: ef31b6228151ed4b27d763ed8d1ddeb4f45dda42d5786f5822ec11e2f391cab5
+      size: 6195697
+      filename: clrmamepro.zip
+    - type: extra-data
+      only-arches:
+        - i386
+      url: https://mamedev.emulab.it/clrmamepro/binaries/cmp4034_32.zip
+      sha256: 0f83ccbdefae39d7526a389f9b52193520d915012cf398b3a908ef117ce2488d
+      size: 5522753
+      filename: clrmamepro.zip
+    - type: script
+      dest-filename: apply_extra
+      commands:
+        - mkdir -p /app/extra/clrmamepro
+        - unzip -qq -d /app/extra/clrmamepro clrmamepro.zip
+        - rm clrmamepro.zip
+    - type: script
+      only-arches:
+        - x86_64
+      dest-filename: clrmamepro
+      commands:
+        - export WINEARCH=win64
+        -
+        # clrmamepro writes it's config files in the current directory
+        # Running files in /app/extra wont work, instead make a copy to
+        # /var/data a.k.a $XDG_DATA_HOME
+        - mkdir -p /var/data/clrmamepro
+        - cp -a /app/extra/clrmamepro /var/data
+        -
+        - wineboot
+        -
+        - wine64 /var/data/clrmamepro/cmpro64.exe "$@"
+    - type: script
+      only-arches:
+        - i386
+      dest-filename: clrmamepro
+      commands:
+        - export WINEARCH=win32
+        -
+        # clrmamepro writes it's config files in the current directory
+        # Running files in /app/extra wont work, instead make a copy to
+        # /var/data a.k.a $XDG_DATA_HOME
+        - mkdir -p /var/data/clrmamepro
+        - cp -a /app/extra/clrmamepro /var/data
+        -
+        - wineboot
+        -
+        - wine /var/data/clrmamepro/cmpro.exe "$@"
+    - type: file
+      path: it.emulab.mamedev.clrmamepro.appdata.xml
+    - type: file
+      path: it.emulab.mamedev.clrmamepro.desktop
+    - type: file
+      path: icons/it.emulab.mamedev.clrmamepro.48x48.png
```

Comment:
Does `clrmamepro` happen to have icons at 128px & 64px? Trying to follow Flathub's packaging guide on icon requirements but if not, no worries.

Reply:
It does not. It does have one for 16x16 but it is entirely broken-looking so not included.